### DISCUSSION TITLE
DockerFile issues with AWS

### DISCUSSION THREAD

**Discussion by ala85** [danny-avila/LibreChat#4962] [Troubleshooting]
### What happened?

When building Docker image to host on AWS as following: 

`docker buildx build --progress=plain --platform linux/amd64 -f Dockerfile -t ACCOUNT.dkr.ecr.eu-north-1.amazonaws.com/app:1.0.0 .`

there are issues as  Cannot find module 'rollup' and similar. Those dependencies are listed as dev dependencies in data-provider json.

### Steps to Reproduce

1. Checkout code as it is
2. run docker buildx build --progress=plain --platform linux/amd64 -f Dockerfile -t YOUR-ACCOUNT.dkr.ecr.eu-north-1.amazonaws.com/app:1.0.0 . to build the image
3. Error should appear

Workaround is to have those as dependencies in data-provider JSON, but feels not proper solution

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
[Dockerfile](https://github.com/danny-avila/LibreChat/blob/51e016ef2c2de7303093f616822dcfff165cea7d/Dockerfile) is building locally and in CI environment so moved this to troubleshooting.
